src/components/search/dto/search-results.dto.ts

```diff
@@ -30,7 +29,6 @@ const publicSearchable = {
   Story,
   LiteracyMaterial,
   Song,
-  Location,
```

Comment:
This is still needed here. Sorry for the confusion. The removal on `searchableAbstracts` was the only thing I was trying to point out.